components/_patterns/01-atoms/02-text/00-headings/_headings.scss

```diff
@@ -8,7 +8,7 @@
 // or, to define new values:
 // @include headings($font-size: 1.4em, $color: $accent, $line-height: 2);
 //
-@mixin headings ($color: $gray-darker, $font-family: $heading, $font-weight: 600, $font-size: 1em, $line-height: 1.5, $margin: 0 0 0.5em, $width: 100%){
+@mixin headings ($color: $header-text, $font-family: $heading, $font-weight: 600, $font-size: 1em, $line-height: 1.5, $margin: 0 0 0.5em, $width: 100%){
```

Comment:
I forgot to mention that I've been prefixing my colors in colors-used with `$color-` so that we can have `$color-headings` and `$font-family-headings` (for font family) and not have to wonder whether the variable `$headings` is for the color, or the font family, or what.

ie:
Snippet from `_colors-used.scss`
```
// Text
$color-text: $near-black;
$color-text-hover: $blue-gray;
$color-text-active: $blue-gray;
$color-heading: $black;
$color-text-placeholder: $blue-gray-light;

// Links
$color-link: $blue-lighter;
$color-link-hover: $blue-light;
```

Snippet from `_font-vars.scss`:
```
$font-family-sentinel: "Sentinel SSm A", "Sentinel SSm B", "Merriweather", serif;
$font-family-gotham: "Gotham SSm A", "Gotham SSm B", "Montserrat", sans-serif;
$font-family-gotham-narrow: "Gotham Narrow SSm A", "Gotham Narrow SSm B", "Montserrat", sans-serif;
```

Snippet from `_fonts-used.scss`
```
$font-family-headings: $font-family-sentinel;
$font-family-text: $font-family-gotham;
```